### ISSUE TITLE
IndexError: index is out of bounds for axis 0 with size XXXX

### ISSUE THREAD

**Issue by fortunto2** [Zulko/moviepy#265]
Hi,

Error 
```
OSError: Error in file /ubuntu/conv/150757/conv/baaadilz/vid20160215102221_010.mp4, At time t=0.11-0.15 seconds, indices wanted: 4806-6805, but len(buffer)=5645
index 5645 is out of bounds for axis 0 with size 5645
```
How i can skip like this video file when rendering?
O solve problem...with many same files
```
[MoviePy] >>>> Building video /ubuntu/conv/150757/film/160360/baaadilz.mp4
[MoviePy] Writing audio in baaadilzTEMP_MPY_wvf_snd.mp3
 73%|███████▎  | 464/634 [00:01<00:00, 427.40it/s]Traceback (most recent call last):
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/io/readers.py", line 188, in get_frame
    result[in_time] = self.buffer[indices]
IndexError: index 5645 is out of bounds for axis 0 with size 5645

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./1job.py", line 246, in <module>
    duration = template.fast.render(allParam, data, videos, audios)
  File "/ubuntu/newconv/template/fast.py", line 400, in render
    ffmpeg_params=ffmpeg_params, threads=8, write_logfile=True)  
  File "<decorator-gen-51>", line 2, in write_videofile
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 54, in requires_duration
    return f(clip, _a, *_k)
  File "<decorator-gen-50>", line 2, in write_videofile
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 137, in use_clip_fps_by_default
    return f(clip, _new_a, *_new_kw)
  File "<decorator-gen-49>", line 2, in write_videofile
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 22, in convert_masks_to_RGB
    return f(clip, _a, *_k)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/video/VideoClip.py", line 330, in write_videofile
    verbose=verbose)
  File "<decorator-gen-73>", line 2, in write_audiofile
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 54, in requires_duration
    return f(clip, _a, *_k)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/AudioClip.py", line 204, in write_audiofile
    verbose=verbose, ffmpeg_params=ffmpeg_params)
  File "<decorator-gen-70>", line 2, in ffmpeg_audiowrite
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 54, in requires_duration
    return f(clip, _a, *_k)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/io/ffmpeg_audiowriter.py", line 161, in ffmpeg_audiowrite
    nbytes= nbytes, fps=fps):
  File "/usr/local/lib/python3.4/dist-packages/tqdm-3.8.0-py3.4.egg/tqdm/_tqdm.py", line 459, in __iter__
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/AudioClip.py", line 81, in generator
    buffersize=chunksize)
  File "<decorator-gen-72>", line 2, in to_soundarray
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 54, in requires_duration
    return f(clip, _a, *_k)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/AudioClip.py", line 127, in to_soundarray
    snd_array = self.get_frame(tt)
  File "<decorator-gen-14>", line 2, in get_frame
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 89, in wrapper
    return f(_new_a, *_new_kw)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/Clip.py", line 95, in get_frame
    return self.make_frame(t)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/AudioClip.py", line 292, in make_frame
    for c,part in zip(self.clips, played_parts)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/AudioClip.py", line 293, in <listcomp>
    if (part is not False) ]
  File "<decorator-gen-14>", line 2, in get_frame
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 89, in wrapper
    return f(_new_a, *_new_kw)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/Clip.py", line 95, in get_frame
    return self.make_frame(t)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/Clip.py", line 136, in <lambda>
    newclip = self.set_make_frame(lambda t: fun(self.get_frame, t))
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/fx/volumex.py", line 18, in <lambda>
    return clip.fl(lambda gf, t: factor \* gf(t),
  File "<decorator-gen-14>", line 2, in get_frame
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 89, in wrapper
    return f(_new_a, *_new_kw)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/Clip.py", line 95, in get_frame
    return self.make_frame(t)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/AudioClip.py", line 292, in make_frame
    for c,part in zip(self.clips, played_parts)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/AudioClip.py", line 293, in <listcomp>
    if (part is not False) ]
  File "<decorator-gen-14>", line 2, in get_frame
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 89, in wrapper
    return f(_new_a, *_new_kw)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/Clip.py", line 95, in get_frame
    return self.make_frame(t)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/AudioClip.py", line 292, in make_frame
    for c,part in zip(self.clips, played_parts)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/AudioClip.py", line 293, in <listcomp>
    if (part is not False) ]
  File "<decorator-gen-14>", line 2, in get_frame
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/decorators.py", line 89, in wrapper
    return f(_new_a, *_new_kw)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/Clip.py", line 95, in get_frame
    return self.make_frame(t)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/io/AudioFileClip.py", line 71, in <lambda>
    self.make_frame =  lambda t: reader.get_frame(t)
  File "/usr/local/lib/python3.4/dist-packages/moviepy-0.2.2.11-py3.4.egg/moviepy/audio/io/readers.py", line 194, in get_frame
    "but len(buffer)=%d\n"%(len(self.buffer))+ str(error))
OSError: Error in file /ubuntu/conv/150757/conv/baaadilz/vid20160215102221_010.mp4, At time t=0.11-0.15 seconds, indices wanted: 4806-6805, but len(buffer)=5645
index 5645 is out of bounds for axis 0 with size 5645
```
[Uploading vid2016
[vid20160215102221_010.mp4.zip](https://github.com/Zulko/moviepy/files/131920/vid20160215102221_010.mp4.zip)
0215102221_010.mp4…]()

**Comment by nickysemenza** [comment]
I'm having this issue too.

**Comment by pchheda** [comment]
Same issue for me as well.

**Comment by anandkpandey1** [comment]
I have the same issue

**Comment by tburrows13** [comment]
Apparently fixed in #377. If you still have this problem, please create a new issue demonstrating the code and media files that cause it.
